lib/colorado_lottery.rb

```diff
@@ -0,0 +1,40 @@
+class ColoradoLottery
+
+  attr_reader :registered_contestants,
+              :winners,
+              :current_contestants,
+              :name
+
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(contestant, game_name)
+    if contestant.game_interests != game_name.name
+      return false
+    elsif contestant.game_interests == game_name.name && contestant.age >= 18
+      true
```

Comment:
the parameter being passed in is an actual game object, not a `game_name`, so it would be best to name it `game` for the sake of avoiding confusion for future you/other developers (so you're not doing something like calling `game_name.name`)

Reply:
`contestant.game_interests` is an array, and `game_name.name` is a string, so they will never be equal-- you can't ever get the true value here. you need to check if the array _includes_ the game name. 

You also don't really need the if/else-- the boolean statement itself will return true/false. 
```
  def interested_and_18?(contestant, game)
      contestant.age >= 18 && contestant.game_interests.include?(game.name)
  end
```